hw_23.09.23/task1.py

```diff
@@ -0,0 +1,21 @@
+import re
+
+
+class Email:
+    def __init__(self, email):
+        self.email = email
+        self.validate()
+
+    def validate(self):
+        correct = r'^[a-zA-Z0-9]+(?:\.[a-zA-Z0-9])*@[a-zA-Z0-9-]+\.[a-zA-Z0-9-]{2,}$'
```

Comment:
Works for "user@email.com", but doesn't work for "user@email.com.ua". This part: `{2,}` might be wrong.